package.json

```diff
@@ -70,5 +74,21 @@
       "/build/",
       "/node_modules/"
     ]
+  },
+  "eslintConfig": {
```

Comment:
I'm fairly certain you aren't actually using eslint here since it doesn't parse flow types. I would do that whole linting thing separately (you can make it but you need to specify our esprima parser and you aren't).